std_msgs/Header header
uint32 ultrasonic1
uint32 ultrasonic2
uint32 ultrasonic3
uint32 ultrasonic4
uint32 ultrasonic5
uint32 ultrasonic6
